app/controllers/general.php

```diff
@@ -615,54 +614,46 @@ function router(App $utopia, Database $dbForConsole, SwooleRequest $swooleReques
         }
 
         if ($logger && $publish) {
-            if ($error->getCode() >= 500 || $error->getCode() === 0) {
```

Comment:
why did we remove this check?

Reply:
Because it would never throw 400 error code, even if we set it to `true` in the exceptions definition.

Here we relay on `publish` having proper value, and this check was moved here https://github.com/appwrite/appwrite/pull/7482/files#diff-89eb85ffb7fda44e8ce1c91a90f72616a3d1bcaff0dbdf79486421df81b26f80R248

Reply:
Then we need to modify the check here.. We should not change how $publish works inside the Exception class..


```php
if ($logger && $publish) {
            if ($publish || $error->getCode() >= 500 || $error->getCode() === 0) {
```

This code should work for our case

Reply:
Partly moved to exception, partly left in general.php